src/Context/GtmContext.php

```diff
@@ -64,6 +64,9 @@ public function dataLayerSettingShouldBe($key, $value) {
    */
   public function getDataLayerSettingShouldMatch($key, $regex) {
     $property_value = $this->getDataLayerValue($key);
+    if (is_bool($property_value)) {
+        $property_value = ($property_value==true) ?'true':'false';
```

Comment:
Please format the syntax with spaces, you can use PHP storm for that via menu Code / Reformat code

Reply:
updated